run_casa_imaging_chunked_example.py

```diff
@@ -0,0 +1,197 @@
+# this_uvh.set_targets(only=['ngc3489','ngc3599','ngc4476'])
+# this_uvh.set_interf_configs(only=['C'])
+# this_uvh.set_line_products(only=['hilores'])
+
+this_pph.set_targets()
+this_pph.set_interf_configs(only=['C'])
+this_pph.set_feather_configs(only=['C+tp'])
+
+# Use boolean flags to set the steps to be performed when the pipeline
+# is called. See descriptions below (but only edit here).
+
+do_staging = True
+do_imaging = True
+do_postprocess = True
+
+##############################################################################
+# Run staging
+##############################################################################
+
+# "Stage" the visibility data. This involves copying the original
+# calibrated measurement set, continuum subtracting (if requested),
+# extraction of requested lines and continuum data, regridding and
+# concatenation into a single measurement set. The overwrite=True flag
+# is needed to ensure that previous runs can be overwritten.
+
+if do_staging:
+    this_uvh.loop_stage_uvdata(do_copy=True, do_contsub=True,
+                               do_extract_line=False, do_extract_cont=False,
+                               do_remove_staging=False, overwrite=True,
+                               strict_config=False)
+
+    this_uvh.loop_stage_uvdata(do_copy=False, do_contsub=False,
+                               do_extract_line=True, do_extract_cont=False,
+                               do_remove_staging=False, overwrite=True,
+                               strict_config=False)
+
+    this_uvh.loop_stage_uvdata(do_copy=False, do_contsub=False,
+                               do_extract_line=False, do_extract_cont=True,
+                               do_remove_staging=False, overwrite=True,
+                               strict_config=False)
+
+    this_uvh.loop_stage_uvdata(do_copy=False, do_contsub=False,
+                               do_extract_line=False, do_extract_cont=False,
+                               do_remove_staging=True, overwrite=True,
+                               strict_config=False)
+
+##############################################################################
+# Step through imaging
+##############################################################################
+
+# Image the concatenated, regridded visibility data. The full loop
+# involves applying any user-supplied clean mask, multiscale imaging,
+# mask generation for the single scale clean, and single scale
+# clean. The individual parts can be turned on or off with flags to
+# the imaging loop call but this call does everything.
+
+if do_imaging:
+
+    # Initialize the ImagingChunkedHandler
+    # Unlike ImageHandler, you must specify the target, configuration and line_name
+    # ImagingChunkedHandler is designed to image individual products that require
+    # chunking to process efficiently.
+    this_imh = ImagingChunkedHandler('ic10ctr', 'C', 'hilores', this_kh,
+                                     chunksize=10,
```

Comment:
Maybe add some guidance on chunksize relative to available RAM?

Reply:
Good point. I'll need to do that upstream.

Right now, this should still default to triggering tclean's subcube mode.